### PR TITLE
browserpass: add brave support

### PR DESCRIPTION
**Pull request by yshym**
### Description

Add support of [Brave browser](https://brave.com/) to browserpass module

### Checklist

- [x] Change is backwards compatible.

- [x] Code formatted with `./format`.

- [ ] Code tested through `nix-shell --pure tests -A run.all`.

- [ ] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

    See [CONTRIBUTING](https://github.com/nix-community/home-manager/blob/master/docs/contributing.adoc#sec-commit-style) for more information and [recent commit messages](https://github.com/nix-community/home-manager/commits/master) for examples.

- If this PR adds a new module

  - [ ] Added myself as module maintainer. See [example](https://github.com/nix-community/home-manager/blob/068ff76a10e95820f886ac46957edcff4e44621d/modules/programs/lesspipe.nix#L6).

  - [ ] Added myself and the module files to `.github/CODEOWNERS`.

### CODE REVIEW

File: tests/modules/programs/browserpass/browserpass.nix [hunk 1]
```diff
@@ -6,20 +6,23 @@ with lib;
...
       for dir in "Google/Chrome" "Chromium" "Vivaldi"; do
         assertFileExists "home-files/Library/Application Support/$dir/policies/managed/com.github.browserpass.native.json"
       done
     '' else ''
-      for dir in "google-chrome" "chromium" "vivaldi"; do
+      for dir in "BraveSoftware/Brave-Browser" "google-chrome" "chromium" "vivaldi"; do
         assertFileExists "home-files/.config/$dir/NativeMessagingHosts/com.github.browserpass.native.json"
+      done
+
+      for dir in "google-chrome" "chromium" "vivaldi"; do
```
Comment (otavio) [inline tests/modules/programs/browserpass/browserpass.nix:25]: Isn't `"BraveSoftware/Brave-Browser"` missing here?

**Review by otavio — CHANGES_REQUESTED** [review]

File: tests/modules/programs/browserpass/browserpass.nix (hunk 1 above)
Comment (yshym) [reply tests/modules/programs/browserpass/browserpass.nix:25]: I just checked and it seems like brave is reading policies just from `/etc/brave/policies` on linux (https://github.com/brave/brave-browser/issues/19052)

Comment (otavio) [reply tests/modules/programs/browserpass/browserpass.nix:25]: So it might be worth adding a comment about it.

**Review by otavio — APPROVED** [review]

**Comment by rycee** [comment]
Thanks for the contribution! And thanks @otavio for the nice reviewing! Merged to master now.

**Pull request merged**